hardware.html

```diff
@@ -1,3 +1,21 @@
+<html>
+  <head>
+    <title>Tamagotchi Architecture</title>
+  </head>
+    
+    
+  <body>
+    <h1>Tamagotchi Hardware</h1>
+    <p>The examination of the very original version of Tamagotchi,to Tamagotchi Plus to the most recent mobile app version reveals that all of the devices are designed and manufactured in the shape of an egg. This makes logical sense given three main reasons: first, “Tama” means an egg in the Japanese language while “gotchi” stands for watch. Then an egg pops up on the screen of the device and a Tama pet hatches from the egg after the owner activates the device. The last but not the least in terms of sensationalizing customers, the egg shape gives more of a smooth shape feeling compared to a rectangular or square shape. This makes the Tamagotchi device more appealing and convenient to carry in the hand.  Moreover, the variety of color and pattern design on the egg shape attracts a young generation of customers.
```

Comment:
All you need to do is add this paragraph in place of the paragraph below the `---` at the top of the file. I never was explicit about that part -- I'm so sorry! You don't need to add the other HTML elements to the page.